src/structures/BaseGuildVoiceChannel.js

```diff
@@ -80,6 +135,45 @@ class BaseGuildVoiceChannel extends GuildChannel {
     );
   }
 
+  /**
+   * Creates an invite to this guild channel.
+   * @param {CreateInviteOptions} [options={}] The options for creating the invite
+   * @returns {Promise<Invite>}
+   * @example
+   * // Create an invite to a channel
+   * channel.createInvite()
+   *   .then(invite => console.log(`Created an invite with a code of ${invite.code}`))
+   *   .catch(console.error);
+   */
+  createInvite(options) {
+    return this.guild.invites.create(this.id, options);
+  }
+
+  /**
+   * Fetches a collection of invites to this guild channel.
+   * Resolves with a collection mapping invites by their codes.
```

Comment:
This is not included in the v14 PR.

Reply:
It's not, but some formatting should be done...